source/source.go

```diff
@@ -147,9 +148,11 @@ func parseTemplate(fqdnTemplate string) (tmpl *template.Template, err error) {
 	if fqdnTemplate == "" {
 		return nil, nil
 	}
-	funcs := template.FuncMap{
-		"trimPrefix": strings.TrimPrefix,
-	}
+
+	funcs := sprig.HermeticTxtFuncMap()
```

Comment:
could we use go template instead here?

extra := template.FuncMap{
    "isIPv6" : isIPv6String
    "isIPv4" : isIPv4String
}

and add them to `funcs` if it's possible

Reply:
this code most likely better to move to 
```
func funcMap() template.FuncMap {}
```

so we could have an isolated code coverage for multiple functions